shapely/tests/legacy/test_clip_by_rect.py

```diff
@@ -1,123 +1,120 @@
-"""

```

Comment:
I'm not sure why it shows this way. the only change in this file besides the rename is the removal of 
`pytestmark = pytest.mark.skipif(geos_version < (3, 5, 0), reason="GEOS 3.5.0 required")` and `from shapely.geos import geos_version`

Reply:
solved via rebasing and separated commits for move and edit